test/Status.Steps.ps1

```diff
@@ -4,39 +4,38 @@ if(!(git config --get user.email)) {
     git config --global core.autocrlf "true"
 }
 
+BeforeScenario {
+    $script:repo = Convert-Path TestDrive:\
+    Push-Location TestDrive:\
+    [Environment]::CurrentDirectory = $repo
+    Remove-Item TestDrive:\* -Recurse -Force
+}
+
+AfterScenario {
+    Pop-Location
+    [Environment]::CurrentDirectory = Convert-Path $Pwd
+}
+
 Given "we have a command ([\w-]+)" {
     param($command)
     $script:command = Get-Command $command -Module PSGit
 }
 
 Given "we are NOT in a repository" {
-    $script:repo = Convert-Path TestDrive:\
-    Push-Location TestDrive:\
-    [Environment]::CurrentDirectory = $repo
-
-    Remove-Item TestDrive:\* -Recurse -Force
+    # Remove-Item TestDrive:\* -Recurse -Force
+    if(gci){ throw "There Are Things Here!" }
```

Comment:
Do you think that this should be a **gci .git** since the statement is just, not in a repo?